apim-operator/deploy/crds/wso2_v1alpha1_security_cr.yaml

```diff
@@ -8,7 +8,4 @@ spec:
   type: Oauth
   certificate: secret-test
   endpoint: https://wso2apim.wso2.svc:9443
-  credentials: secret-test-certificate
-  issuer: https://localhost:9443/oauth2/test
-  alias: my-alias
-  audience: https://localhost:9443/oauth2/test
\ No newline at end of file
+  credentials: secret-test-certificate
```

Comment:
Same as above. Add empty new line

Reply:
File was moved to scenarios and it's fixed in the latest code